{vis}[GCCcore/12.3.0] Xvfb v21.1.8

stefan-wolfsheimer opened the pull request:
(created using `eb --new-pr`)
depends on #18544 #18449 #18526

branfosj commented on easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:
```diff
@@ -0,0 +1,123 @@
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['699d1a62012035b1461c7f8e3f05a51c8bd6f28f348983249fb89bbff7309b47'],
+    }),
+    ('font-util', '1.3.3', {
```
There is a 1.4.0 available (https://www.x.org/archive/individual/font/).

branfosj commented on easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:
```diff
@@ -0,0 +1,123 @@
+    ('font-util', '1.3.3', {
+        'checksums': ['2094dd4a1ca63a61deb101d2dc618682d6e287cdbe09679502223ac445d277dc'],
+    }),
+    ('font-misc-misc', '1.1.2', {
```
There is a 1.1.3 available (https://www.x.org/archive/individual/font/).

branfosj commented on easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:
```diff
@@ -0,0 +1,123 @@
+        'source_urls': ['https://www.x.org/archive/individual/app/'],
+        'checksums': ['b216a2c8c0eab83f3dc4a3d5ee2bdf7827b30e49c8907035d0f222138eca0987'],
+    }),
+    ('xkeyboard-config', '2.37', {
```
There is a 2.39 available (https://www.x.org/archive/individual/data/xkeyboard-config/),

branfosj commented on easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb:
```diff
@@ -0,0 +1,123 @@
+easyblock = 'Bundle'
+
+name = 'Xvfb'
+version = '21.1.6'
```
There is a 21.1.8 available (https://www.x.org/releases/individual/xserver/).

stefan-wolfsheimer replied:
Thanks for the review. I will add more recent versions.

stefan-wolfsheimer commented:
I added the new file (v21.1.8) with `--update-pr` and deleted the old version (v21.1.6) with `git rm`
This generated some clutter in the logs. Should I create a fresh pr with the correct version?

casparvl commented:
> I added the new file (v21.1.8) with `--update-pr` and deleted the old version (v21.1.6) with `git rm` This generated some clutter in the logs. Should I create a fresh pr with the correct version?

To me, the current one is fine. I've seen way worse clutter than this :)

casparvl commented:
Test report by @casparvl
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/ff822b3cff8ca949a5aa443d802067cd for a full test report.

casparvl commented:
Going in, thanks @stefan-wolfsheimer!
